app/views/wikis/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1>New Wiki</h1>
+
+ <div class="row">
+   <div class="col-md-8">
+     <%= form_for @wiki do |f| %>
```

Comment:
Since you have the same form for the edit page and the new page, it makes sense to move it to a partial, that way when you add things to the form you only have to do it once. Do this in the next PR.